scss/_patterns_notifications.scss

```diff
@@ -1,62 +1,146 @@
 @import 'settings';
 $notification-content-vert-padding: $spv-inner--small;
 $notification-content-icon-space: 2 * $sph-inner + map-get($icon-sizes, default);
-$notification-icon-vert-offset: 0.5 * (map-get($line-heights, default-text) - map-get($icon-sizes, default));
+$notification-icon-vert-offset: 0.6rem;
+$notification-text-margin-bottom: 0.6rem;
 
 // Notification style patterns
 @mixin vf-p-notification {
   // The mixin for basic notification styling
   %vf-notification {
     @extend %vf-is-bordered;
 
+    background-position: $sph-inner $notification-icon-vert-offset;
```

Comment:
I think we should set background color to white as well, to make sure it works correctly on other backgrounds. It seems that the old version was doing it.